Assets/Code/Behaviours/GenerateCity.cs

```diff
@@ -27,10 +44,16 @@ void Start()
                 var cell = _map.GetMapCell(row, col);
 
                 if (cell != null)
-                {
-                    if (cell.MapCellType == Map.MapCellType.Road)
-                    {
-                       
+                {
+                    if (cell.MapCellType == MapCellType.Road)
+                    {
+                        if (Random.Range(0f, 1f) < CreatePedestrainProbability)
+                        {
+                            var newPedestrian = Instantiate(pedestrian, GetNewPedestrianPosition(cell), Quaternion.identity);

```

Comment:
Also can be done on some other iteration.

But i need to clean up the code in this class (it was created with a lot of things, just to make it work quickly and dirty).
So, i think we should also extract the pedestrian creation into another behavior that has binding to this behavior (same way i did with the car positioning).